Jenkinsfile

```diff
@@ -2,9 +2,10 @@ pipeline {
     agent {
         kubernetes {
             label "builder-pod-${UUID.randomUUID().toString()}"
+            cloud "go-demo-4-build"
             defaultContainer 'jnlp'
```

Comment:
This is exact copy of `CiJenkinsfile`, it just appears as a code change because there used to be a `Jenkinsfile` as well.